src/plugins/data/public/search/search_service.ts

```diff
@@ -171,6 +172,13 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
           return new SearchSource({}, searchSourceDependencies);
         },
       },
+      searchParams: {
```

Comment:
I think it would be really simple to add this function as a utility, passing in `core` from the caller.
I don't see any reason to have this as part of the contract, as it's only used to migrate old requests to the new search service format, and won't be a part of the API long term

Reply:
@liza-mae np, will move it into static contract.  My idea was to bind the required dependencies...

Reply:
Wrong liza :)

Reply:
=)